app/web_framework/cluster_base_handler.py

```diff
@@ -62,6 +66,14 @@ def _handle_request_exception(self, ex):
         self.set_status(status_code)
         self.finish()
 
+    def _get_pagination_params(self):
```

Comment:
Imo this method shouldn't be tested in isolation in the first place. I don't really care if this method is behaving properly, I care if the pagination functionality is behaving properly. This method is just a means to an end, so don't test it. (It's also private, which is an indication that it's not a good thing to test).

A better test would be a functional test that passes various `offset`/`limit` combinations to cluster runner and verifies that we get back the set of builds we expect. Don't test _functions_, test _functionality_.

I know that flies in the face of the concept of unit testing, and some people on the team disagree with me. So do what you want.